app/models/seance.rb

```diff
@@ -7,6 +7,11 @@ class Seance < ApplicationRecord
   validates :price, numericality: { only_decimal: true, greater_than: 0 }
   validate :used?
 
+  scope :week, lambda {
```

Comment:
`:this_week` would be a better name for this scope, 
much clearer what it does when used in the controller